### PR TITLE
Add `next` to peerDependencies

### LINKED ISSUE #43: Make next a peer dep
It would let developers use a canary version of `next` if desired.

### PR DESCRIPTION
**Pull request by cemremengu**
Closes #43

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,10 +38,12 @@
     "url": "https://github.com/fastify/fastify-react/issues"
   },
   "homepage": "https://github.com/fastify/fastify-react#readme",
-  "dependencies": {
-    "fastify-plugin": "^1.2.1",
+  "peerDependencies": {
     "next": "^7.0.2"
```
Comment (mcollina) [inline package.json]: we should also add `next`  to devdep.

Comment (cemremengu) [reply package.json]: oops

**Comment by mcollina** [comment]
The readme instructions would need to updated as well.

```
npm i fastify-react next
```

**Comment by cemremengu** [comment]
Let's wait for #47 to land both

**Pull request merged**